functions/supplier/cdc-debezium-supplier2/src/test/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.mongo.MongoAutoConfiguration
```

Comment:
I don't think we need this since we do that in the code via `@EnableAutoConfiguration`

Reply:
resolved